scripts/lint-style.py

```diff
@@ -65,20 +68,12 @@
     for exline in f:
         filename, _, _, _, _, errno, *extra = exline.split()
         path = ROOT_DIR / filename
-        if errno == "ERR_COP":
```

Comment:
This removes some variants; a later commit implements parsing *all* exception types.
(I didn't feel like editing history again to edit this part out.)